src/ring_theory/algebraic.lean

```diff
@@ -74,6 +74,12 @@ variables (R : Type u) {A : Type v} [comm_ring R] [nontrivial R] [ring A] [algeb
 lemma is_integral.is_algebraic {x : A} (h : is_integral R x) : is_algebraic R x :=
 by { rcases h with ⟨p, hp, hpx⟩, exact ⟨p, hp.ne_zero, hpx⟩ }
 
+variables {R}
+
+/-- An element of `R` is algebraic, when viewed as an element of the `R`-algebra `A`. -/
+lemma is_algebraic_of_mem (a : R) : is_algebraic R (algebra_map R A a) :=
```

Comment:
```suggestion
lemma is_algebraic_algebra_map (a : R) : is_algebraic R (algebra_map R A a) :=
```
Might be a better name, since there is no \mem anywhere in this statement